app/models/game.rb

```diff
@@ -1,3 +1,46 @@
 class Game < ActiveRecord::Base
-	belongs_to :name
+	belongs_to :type
+	after_create :populate_board!
+
+	def populate_board!
+		my_game = game.create
```

Comment:
Since this happens in an `after_create`. The game is already created.

So you could add `has_many :pieces`

And then use `pieces.create' :smile: